suite_cam6.xml

```diff
@@ -0,0 +1,363 @@
+      <scheme>blj_diag_output</scheme>                <!-- Calculate SGO diagnostic variables, and output them to history file -->
+      <scheme>theta_calc</scheme>                     <!-- Calculate potential temperature (managed by registry/framework?) -->
+      <scheme>virtem</scheme>                         <!-- Calculate virtual temperature (managed by registry/framework?) -->
+      <scheme>calc_ustar</scheme>                     <!-- Calculate friction velocity -->
+      <scheme>calc_obklen</scheme>                    <!-- Calculate Obhukov length -->
+      <scheme>pbl_diag_calc</scheme>                  <!-- Calculate PBL diagnositc variables, if do_pbl_diags is False -->
+      <scheme>dse_top_calc</scheme>                   <!-- Calculate top-level dry static energy, but only if do_molec_diff is True and WACCM-X is False -->
+      <scheme>compute_molec_diff_wet</scheme>         <!-- Calculate molecular diffusion of wet atmospheric constituents, if do_molec_diff is True -->
+      <scheme>compute_vdiff_wet</scheme>              <!-- Calculate the vertical (eddy) diffusion of wet atmospheric constituents -->
+      <scheme>compute_molec_diff_dry</scheme>         <!-- Calculate molecular diffusion of dry atmospheric constituents, if do_molec_diff is True -->
+      <scheme>compute_vdiff_dry</scheme>              <!-- Calculate the vertical (eddy) diffusion of dry atmospheric constituents -->
+      <scheme>aero_srf_flx_add</scheme>               <!-- Add surface aerosol fluxes to bottom model layer, but only if prog_modal_aero is True -->
+      <scheme>diff_flux_diag</scheme>                 <!-- Calculate diffusion variable diagnostics, if do_pbl_diags is False (is that correct?) -->
+      <scheme>diff_flux_tend_dry</scheme>             <!-- Calculate vertical diffusion tendencies for "dry" quantities, and send them back to host model -->
+      <scheme>set_wet_to_dry</scheme>                 <!-- Convert mixing ratios from air + water vapor to dry air -->
+      <scheme>pbl_diag_calc</scheme>                  <!-- Calculate PBL diagnositc variables, if do_pbl_diags is False -->
+      <scheme>diff_mass_check</scheme>                <!-- Check mass conservation after vertical diffusion, and send error messages if check fails -->
+      <scheme>pbl_output</scheme>                     <!-- Write PBL diagnostic variables to history files -->
+      <!-- **************************** -->
+      <scheme>rayleigh_friction_tend</scheme>         <!-- Calculate Rayleigh friction tendency -->
+      <scheme>check_energy_chng</scheme>              <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_tracers_chng</scheme>             <!-- Global tracer mass checker that kills model if error is large enough. -->
+      <!-- Aerosol dry deposition -->
+      <!-- **************************** -->
```

Comment:
I believe the ACOM group is doing a whole aerosol project that they are targeting to run using CCPP (part of Musica).  Would it replace this whole section?

Reply:
I think these are different functions. Musica would create the aerosols but it is other physics processes that remove them from the atmosphere. That is what is going on here.